src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -194,17 +220,17 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         return winrt::hstring{ str };
     }
 
-    winrt::hstring CopyTextArgs::GenerateName() const
+    winrt::hstring CopyTextArgs::GenerateName(bool localized) const

```

Comment:
Reading this makes me realize that a localization system should probably pass the context of the localization (i.e. the locale) as a parameter to make the translation function pure. I should probably do the same for Edit...